src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorModelTests.swift

```diff
@@ -17,20 +17,35 @@
 // under the License.
 //
 
-@testable import ENA
-import Foundation
 import XCTest
+@testable import ENA
 
-// swiftlint:disable:next type_body_length
 class ExposureSubmissionCoordinatorModelTests: XCTestCase {
 
+	// default provider for a static app configuration
+	let configProvider = CachedAppConfiguration(client: CachingHTTPClientMock(), store: MockTestStore())
+
+	override func setUpWithError() throws {
+		// uses a common database!
```

Comment:
That does this comment mean? Is it a todo?

Reply:
Just a very simple hint to the common database file. There is no need to have a property or even singleton for `MockTestStore`. Normally you would assume that changes to that store instance would only affect it and not also the stores in the tests. 

This could be elaborated in more detail but I just didn't do it.

Reply:
I changed the comment. It's updated here soon.